### PR TITLE
Compile flatbuffers for Rust in build.rs

### PR DESCRIPTION
**Pull request by nazar-pc**
`flatbuffers` file is now auto-generated during compilation of `mediasoup-sys`, imports were changed accordingly, also `fbs` was added to `worker/Cargo.toml` since otherwise it'll fail to publish.

### CODE REVIEW

File: worker/build.rs [hunk 1]
```diff
@@ -16,6 +16,35 @@ fn main() {
...
+                maybe_entry
+                    .map(|entry| {
+                        let path = entry.path();
+                        if path.extension() == Some("fbs".as_ref()) {
+                            Some(path)
+                        } else {
+                            None
+                        }
+                    })
+                    .transpose()
```
Comment (ibc) [inline worker/build.rs:34]: What does transport() do?

File: rust/src/fbs.rs
Comment (ibc) [inline rust/src/fbs.rs:1]: Assuming this fbs.es file is being generated on compile time, is it gitignored?

File: worker/build.rs [hunk 2]
```diff
@@ -16,6 +16,35 @@ fn main() {
...
+            })
+            .collect::<Result<Vec<_>, _>>()
+            .expect("Failed to collect flatbuffers files"),
+    )
+    .expect("Failed to translate flatbuffers files");
+
+    fs::write(
+        format!("{out_dir}/fbs.rs"),
+        planus_codegen::generate_rust(&flatbuffers_declarations)
```
Comment (ibc) [inline worker/build.rs:43]: That's all? No need to tricks in generated fbs.ts file to make clippy happy and so on?

File: worker/build.rs (hunk 1 above)
Comment (nazar-pc) [reply worker/build.rs:34]: https://doc.rust-lang.org/stable/std/result/enum.Result.html#method.transpose

File: worker/build.rs (hunk 2 above)
Comment (nazar-pc) [reply worker/build.rs:43]: I was also a bit surprised given the fact that we do literal `include!()`, but apparently Rust tracks where source code comes from and from path it is clear that file is auto-generated and it ignores all warnings from there :man_shrugging:

I'm not sure why that happens and lazy to search, but it works out for well for us here :slightly_smiling_face:

File: rust/src/fbs.rs
Comment (nazar-pc) [reply rust/src/fbs.rs:1]: Nope, it doesn't exist in source tree at all, we write it into `OUT_DIR` that is specifically designed for that purpose just like the rest of stuff, see `build.rs`.

Read notes on https://doc.rust-lang.org/cargo/reference/environment-variables.html that contain `OUT_DIR` for details, there are a few of them there.

**Review by ibc — APPROVED** [review]

**Review by jmillan — COMMENTED** [review]
:+1:

**Pull request merged**